init/gravitino/gravitino.conf

```diff
@@ -67,3 +67,4 @@ gravitino.auxService.iceberg-rest.warehouse = hdfs://hive:9000/user/iceberg/ware
 gravitino.auxService.iceberg-rest.jdbc.user = mysql
 gravitino.auxService.iceberg-rest.jdbc.password = mysql
 gravitino.auxService.iceberg-rest.jdbc-driver = com.mysql.cj.jdbc.Driver
+gravitino.auxService.iceberg-rest.catalog-backend-name = catalog_iceberg
```

Comment:
let REST catalog use the same catalog name with Gravitino Iceberg catalog, so they could query same table. cc @jerqi